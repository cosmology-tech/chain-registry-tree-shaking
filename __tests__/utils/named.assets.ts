import { agoric } from 'interchain-assets';

it('list', () => {
    // const result = agoric.map(asset => asset.name);
    const result = agoric.default.assets.map(asset => asset.name);
    expect(result).toMatchSnapshot();
})
