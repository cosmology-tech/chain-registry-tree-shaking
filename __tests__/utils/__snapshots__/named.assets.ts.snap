// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list 1`] = `
[
  "Archway",
  "Axelar",
  "Cosmos Hub Atom",
  "Crescent",
  "Bonded Crescent",
  "Evmos",
  "Neokingdom DAO",
  "Teledisko DAO",
  "Crowdpunk DAO",
  "Graviton",
  "Kujira",
  "Stake",
  "Frienzies",
  "Flix",
  "Osmosis",
  "Ion DAO",
  "Persistence",
  "Secret Network",
  "Stride",
  "Umee",
  "Archway",
  "Axelar",
  "Cosmos Hub Atom",
  "Crescent",
  "Bonded Crescent",
  "Evmos",
  "Neokingdom DAO",
  "Teledisko DAO",
  "Crowdpunk DAO",
  "Graviton",
  "Kujira",
  "Stake",
  "Frienzies",
  "Flix",
  "Osmosis",
  "Ion DAO",
  "Persistence",
  "Secret Network",
  "Stride",
  "Umee",
]
`;
